specs/init.js

```diff
@@ -3,7 +3,7 @@ import thunk from 'redux-thunk';
 import initLocalStorageMock from './mock/localStorage';
 
 // we need enough time to test contract watch functionality
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 200000;
```

Comment:
Please correct the specs/redux/ducks/settings/tokensActions.spec.js tests so that they fit within the allotted time.
https://travis-ci.org/ChronoBank/ChronoMint/builds/211879277